test/e2e/servicebindingrequest_test.go

```diff
@@ -556,13 +556,15 @@ func serviceBindingRequestTest(
 	}, 120*time.Second, 2*time.Second)
 
 	// after deletion, deployment should not contain envFrom directive anymore
+	t.Log("Deployment should not contain envFrom directive")
 	require.Eventually(t, func() bool {
 		t.Logf("Inspecting deployment '%s'", deploymentNamespacedName)
 		_, err := assertDeploymentEnvFrom(todoCtx, f, deploymentNamespacedName, sbrName)
 		if err != nil {
-			t.Log("expect deployment not to be carrying envFrom directive")
+			t.Log("Expect deployment not to be carrying envFrom directive")
```

Comment:
Can we use t.Logf and print the err as well

Reply:
Here we are expecting an error, if we get an error the loop will return true so printing an error msg won't be of any use.